packages/csv-parser-price/src/main.js

```diff
@@ -183,9 +183,11 @@ export default class CsvParserPrice {
 // Easiest way to wrap the getCustomTypeDefinition in the memoize method
 CsvParserPrice.prototype.getCustomTypeDefinition = memoize(
   function _getCustomTypeDefinition (customTypeKey) {
-    const getTypeByKeyUri = createRequestBuilder().types
+    const getTypeByKeyUri = createRequestBuilder({
```

Comment:
So what do you suggest? Because I don't know how we can pass and not pass the projectkey to the `.build()` method at the same time